### PR TITLE
docs: Add All Contributors to the project

### PR DESCRIPTION
**Pull request by matchai**
#### Description

- Add All Contributors section to `README.md`
	- Add all past contributors
- Add `@all-contributors` to the project

#### Motivation and Context

To be able to recognize all the amazing contributors to starship, we will be adding All Contributors. 💕 

To start off we're adding all existing contributors. When we get around to creating `CONTRIBUTING.md`, we will document how future contributors can add themselves to All Contributors.

Did I miss anyone's responsibilities within the project?

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        "ideas",
+        "maintenance"
+      ]
+    },
+    {
+      "login": "Snuggle",
+      "name": "​Snuggle",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/26250962?v=4",
+      "profile": "https://about.snuggi.es",
+      "contributions": [
```
Comment (johnletey) [inline .all-contributorsrc]: @Snuggle reviewed both #40 and #42

**Comment by Snuggle** [comment]
I believe a little bit of design, too, in #2, #3, #4 and #49. 😊💕

File: .all-contributorsrc [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      ]
+    },
+    {
+      "login": "sirMerr",
+      "name": "Tiffany Le-Nguyen",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/11183523?v=4",
+      "profile": "https://github.com/sirMerr",
+      "contributions": [
+        "ideas",
+        "maintenance"
```
Comment (sirMerr) [inline .all-contributorsrc]: I've reviewed and created/updated docs ^^.

Comment (matchai) [reply .all-contributorsrc]: I'll contributions re: PR and issue templates as "Maintenance".
We'll be creating a website with full docs soon, so there'll be work to do there. 🙂

**Pull request merged**